MbDotNet/Models/Imposters/HttpImposter.cs

```diff
@@ -1,17 +1,26 @@
 ﻿using System.Collections.Generic;
+using MbDotNet.Models.Responses.Fields;
 using MbDotNet.Models.Stubs;
 using Newtonsoft.Json;
 
 namespace MbDotNet.Models.Imposters
 {
-    public class HttpImposter : Imposter 
+    public class HttpImposter : Imposter
     {
         [JsonProperty("stubs")]
         public ICollection<HttpStub> Stubs { get; private set; }
 
-        public HttpImposter(int? port, string name, bool recordRequests = false) : base(port, Enums.Protocol.Http, name, recordRequests)
+        /// <summary>
+        /// Optional default response that imposter sends back if no predicate matches a request
+        /// </summary>
+        [JsonProperty("defaultResponse", NullValueHandling = NullValueHandling.Ignore)]
+        public HttpResponseFields DefaultResponse { get; set; }

```

Comment:
Please make this a `private set` on each imposter type to be consistent with the other properties.